### ISSUE TITLE
Proxy issue when downloading docs (but no problem loading docsets index)

### ISSUE THREAD

**Issue by cccaballero** [zealdocs/zeal#666]
Hi, using Zeal 3.1 on Ubuntu 16.04 behind a proxy, it downloads fine the docsets index, but shows an error window with the info "Error downloading http://newyork.kapeli.com/feeds/SciPy.tgz - server replied: Forbidden" when it tries to download the doc package.

**Comment by trollixx** [comment]
Maybe your proxy server requires authentication?

**Comment by cccaballero** [comment]
@trollixx yes, the proxy needs authentication and is properly configured (if not it would not download the docsets index)

**Comment by trollixx** [comment]
Does it work when you use system's proxy configuration?

**Comment by cccaballero** [comment]
With the system proxy configured (and no zeal proxy config) it shows an alert dialog saying "Host go.zealdocs.org not found"
With the system proxy and the zeal proxy configured, it throws the "forbidden" error.

**Comment by trollixx** [comment]
This is all a bit strange, I'll try to setup a test environment and with a proxy requiring authentication.

**Comment by trollixx** [comment]
I was able to reproduce the problem. Feels like a bug in Qt since SSL works via proxy if auth is disabled, and unsecured traffic works in either case.

**Comment by trollixx** [comment]
[QTBUG-55960](https://bugreports.qt.io/browse/QTBUG-55960) must be the reason.
